Package.swift

```diff
@@ -9,6 +9,9 @@ let package = Package(
     .library(
       name: "Base64Kit",
       targets: ["Base64Kit"]),
+    .library(
+      name: "Base32Kit",
+      targets: ["Base32Kit"])
```

Comment:
I thought about `BaseKit` but I suspect that it might be confusing. One could think that it is something like a `Commons` package in other language like Java. Just stuff that is _commonly_ needed or something. Even though it is a bit more verbose, we might want something like `BaseEncodingKit`?

Reply:
> do you have arguments for one or the other on that?

No. My gut tells me it would be better to have a single target. Naming will be the challenge 😄

Reply:
@jenslauterbach Okay. Then let's just merge everything into Base64Kit first, and we can do a big renaming pr later.

Reply:
@fabianfett Great. I am will make the required changes.